src/modules/shared/components/Tabs/ScrollableTabs/ScrollableTabs.tsx

```diff
@@ -0,0 +1,171 @@
+import { Tabs, TabsProps } from '@meemoo/react-components';
+import clsx from 'clsx';
+import React, { FC, useCallback, useEffect, useRef, useState } from 'react';
+
+import styles from './ScrollableTabs.module.scss';
+
+const ScrollableTabs: FC<TabsProps> = (props) => {
+	const { tabs: items } = props;
+
+	/**
+	 * Hooks
+	 */
+	const hasInitialised = useRef(false);
+	const scrollContainerRef = useRef<HTMLDivElement | null>(null);
+	const tabsRef = useRef<Element | null>(null);
+	const [activeEl, setActiveEl] = useState<Element | null>(null);
+	const [tabsHeight, setTabsHeight] = useState(0);
+	const [showLeftGradient, setShowLeftGradient] = useState(false);
+	const [showRightGradient, setShowRightGradient] = useState(false);
+
+	// Hide horizontal scrollbar
+	useEffect(() => {
+		if (scrollContainerRef.current) {
+			const tabsEl = scrollContainerRef.current.querySelector('.c-tabs');
+
+			if (tabsEl) {
+				tabsRef.current = tabsEl;
+				setTabsHeight(tabsEl.clientHeight);
+			}
+		}
+	}, []);
+
+	// Set active element
+	useEffect(() => {
+		if (tabsRef.current && items.length) {
+			setActiveEl(tabsRef.current.querySelector('.c-tab--active'));
+		}
+	}, [items]);
```

Comment:
Ben eerder fan van een `onClick` handler op `Tabs` dan gebruik te maken van een hook.

Reply:
Deze gaat enkel bijhouden welke tab actief is omdat deze altijd initieel in view wordt gescrolled.
onClick functionaliteit zit in de Tabs component zelfs niet hier.